keps/sig-network/2079-network-policy-port-range/README.md

```diff
@@ -232,20 +232,42 @@ _This section must be completed when targeting alpha to a release._
 
 * **Can the feature be disabled once it has been enabled (i.e. can we roll back
   the enablement)?**
-  Yes, but CNIs relying on the new field wont recognize it anymore
+  Yes, but disabling the feature gate and updating a Network Policy with existing 
+  EndPort field will remove this field.
+
+  Rolling back the Kubernetes API Server that does not have this field
+  will make the field not be returned anymore on GET operations,
+  so CNIs relying on the new field wont recognize it anymore.
 
 * **What happens if we reenable the feature if it was previously rolled back?**
   Nothing. Just need to check if the data is persisted in `etcd` after the 
-  feature is disabled and reenabled or if the data is missed
+  feature is disabled and reenabled. If the object "NetworkPolicy" is updated
+  during this time window between disabling and re-enabling the EndPort field 
+  from this object might be removed by PrepareForUpdate function.
```

Comment:
what do you mean by "might" be removed? Is there a certain scenario where this would occur or is it unknown whether or not this would / should occur?

Reply:
was not known, but as per previous comment it's also not desired, so will remove this from here, and implement the change to drop only if this is a new field, and not present on the old netpol object.

This is probably going also to take some more time as it's good now to implement the tests about enable->disable->enable FG

Reply:
Let's remove this part - this will will not be removed.